core-services/src/main/java/org/zalando/nakadi/service/NakadiAvroSchemaService.java

```diff
@@ -0,0 +1,48 @@
+package org.zalando.nakadi.service;
+
+import org.apache.avro.Schema;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class NakadiAvroSchemaService {
+
+    public static final class VersionedAvroSchema {
+        private final Schema avroSchema;
+        private final String version;
+
+        public VersionedAvroSchema(final Schema avroSchema, final String version) {
+            this.avroSchema = avroSchema;
+            this.version = version;
+        }
+
+        public Schema getAvroSchema() {
+            return avroSchema;
+        }
+
+        public String getVersion() {
+            return version;
+        }
+    }
+
+    private final LocalSchemaRegistry localSchemaRegistry;
+
+    @Autowired
+    public NakadiAvroSchemaService(final LocalSchemaRegistry localSchemaRegistry) {
+        this.localSchemaRegistry = localSchemaRegistry;
+    }
+
+    public List<VersionedAvroSchema> getNakadiAvroSchemas(final String schemaName) {
+        final var schemas = localSchemaRegistry.getEventTypeSchemaVersions(schemaName);
+        return schemas.entrySet().stream()
+                .map(entry -> new VersionedAvroSchema(entry.getValue(), entry.getKey()))
+                .collect(Collectors.toList());
```

Comment:
The list is probably ordered by version, given the underlying implementation.  Should we make the ordering part of the API contract?

Reply:
Yeah, I'm just wondering how the users are going to work with it and what would be their expectation.